src/sp/it/pl/audio/Player.java

```diff
@@ -85,9 +85,9 @@
 	public static final InOutput<Item> anySelected = new InOutput<>(UUID.fromString("1a01ca96-2e60-426e-831d-93b24605595f"), "Selected anywhere", Item.class);
 
 	@IsConfig(name = "Remember playback state", info = "Continue last remembered playback when application starts.")
-	public static boolean continuePlaybackOnStart = true;
+	public static boolean continuePlaybackOnStart = false;
```

Comment:
What is the reasoning behind changing the default?

Reply:
You don't expect your player to blast you down upon start by default, especially since this takes some time to start.

Reply:
If you close the player with PAUSED or STOPPED, it will continue PAUSED or STOPPED. Perhaps it is only me, but preserving the state seems like a good default to me. You also turned off both preservePlabackState and startPaused, which already overrides the former. Was that necessary?

Reply:
now that i think of it, this should be one multi-way config instead of two booleans. Either pick up state or start paused or - what is startPaused even for? will it start playing if both are false?

But it is simply expected behaviour for me as user that it doesn't continue playback on start unless told so. Every player I've used defaulted to start silent and most then had an option to keep playback state.

Reply:
Ok